spec/apps/bootstrap-rejects/bootstrap-rejects.app.js

```diff
@@ -1,22 +0,0 @@
-export let wasBootstrapped = false;
```

Comment:
I renamed the `spec/apps` directory to `spec/child-apps` and then modified all of the apps slightly to account for the new karma/saucelabs/jspm testing environment. So that's why it looks like all the tests were deleted.